cpp/splinepy/py/py_multi_patch.hpp

```diff
@@ -1808,6 +1808,68 @@ class PyMultiPatch {
 
   /// @brief Gets list of fields
   py::list GetFields() { return field_list_; }
+
+  /// @brief Get summed number of all control points
+  int GetNumberOfControlPoints() {
+    // Init return value
+    int n_control_points{};
+    for (std::size_t i_patch{}; i_patch < core_patches_.size(); i_patch++) {
+      n_control_points +=
+          core_patches_[i_patch]->SplinepyNumberOfControlPoints();
+    }
+    return n_control_points;
+  }
+
+  /// @brief Get summed number of all control points
+  py::array_t<int> GetControlPointOffsets() {
+    // Init return value
```

Comment:
we can probably save those too?

Reply:
I would just recompute. It's rarely used and cheap to compute